libraries/BLE/examples/notification/notification.ino

```diff
@@ -0,0 +1,81 @@
+

```

Comment:
Ok, thanks for the clarification. In the future could you please add a comment in the sketch to indicate it is for testing only.